commit 3ddd07cfbe4ecfce12f91fe5623eec5c89dc3017
Author:     Yan Cheng <58191769+yanchengnv@users.noreply.github.com>
AuthorDate: Mon Sep 18 18:27:45 2023 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Mon Sep 18 15:27:45 2023 -0700

    Support secure task in CCWF and comm improvement (#1999)
    
    * added secure task for ccwf; use memview for efficiency
    
    * change to use core_cell for fire_and_foget
    
    * remove unused import
    
    * added private_p2p arg
    
    * fix imports
    
    * disable test_create_server temporarily
    
    * fix import
    
    * allow multiple concurrent job download; test huge workspace download
    
    * polish
    
    * changed to use event to print download progress
    
    * fix fire_and_forget
    
    * support destination folder for job download
    
    * fix filter processing
    
    * address PR findings
---
 nvflare/private/fed/server/job_runner.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/job_runner.py b/nvflare/private/fed/server/job_runner.py
index 0d733db2..74a5bbad 100644
--- a/nvflare/private/fed/server/job_runner.py
+++ b/nvflare/private/fed/server/job_runner.py
@@ -389,7 +389,7 @@ class JobRunner(FLComponent):
             output_file = os.path.join(td, "workspace")
             zip_directory_to_file(run_dir, "", output_file)
             job_manager.save_workspace(job_id, output_file, fl_ctx)
-            self.log_info(fl_ctx, f"Workspace zipped to {output_file}")
+            self.log_debug(fl_ctx, f"Workspace zipped to {output_file}")
         shutil.rmtree(run_dir)
 
     def run(self, fl_ctx: FLContext):
